tests/core/data/test_filter_sql_table_data_node.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, foo=None, bar=None, *args, **kwargs):
 
 
 class TestFilterSQLTableDataNode:
-    __pandas_properties = [
+    __sql_properties = [
```

Comment:
Oh, are we using the `__pandas_properties` or the `__sql_properties` in this file at all?
Is this a redundant variable?

Reply:
ah good point, don't think we do! let me remove them